config/parse.go

```diff
@@ -276,6 +292,57 @@ func File(f string) (*Config, error) {
 	return Bytes(dat)
 }
 
+func findConfigFiles(path string) ([]string, error) {
+	confLog.WithField("path", path).Debugf("Reading configuration files from %s", path)
+	configFiles := make([]string, 0)
+	err := filepath.Walk(path, func(path string, info os.FileInfo, err error) error {
+		if !info.IsDir() && len(strings.Split(path, ".")) > 1 &&
+			strings.ToLower(strings.Split(path, ".")[1]) == "json" {
```

Comment:
This will fail if the file is name "foo.bar.zoo.json" unless I'm reading this wrong. 

Can't we just check that len(strings) > len(".json") then check if strings[len(strings)-len(".json"):] == ".json" or something like that?

Reply:
Hmm, also, walk doesn't follow symbolic links? That's.... problematic and downright stupid, but we can live with that _for now_.

Reply:
I've resolved the original comment, but walk still works as walk does. We might look into that in the future as you say.